components/automate-ui/src/app/modules/infra-proxy/environments/environments.component.html

```diff
@@ -0,0 +1,17 @@
+<section class="environments">
+  <chef-loading-spinner *ngIf="environmentsListLoading" size="50"></chef-loading-spinner>
+  <ng-container *ngIf="!environmentsListLoading">
+    <chef-table>
+      <chef-thead>
+        <chef-tr>
+          <chef-th>Environments Name</chef-th>
```

Comment:
I suspect this should be the singular "Environment Name". @susanev or @jonong1972 can you confirm?